src/app/posts/comments/comment.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+import {HttpClient, HttpErrorResponse, HttpParams} from "@angular/common/http";
+import {Observable, throwError} from "rxjs";
+import {Comment} from "./comment.interface";
+import {catchError, retry} from "rxjs/operators";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CommentService {
+
+  static readonly API_URL = "http://localhost:3000/comments?_sort=id&_order=desc"
+
+  constructor(private http: HttpClient) { }
+
+  getComments(postId:number): Observable<Comment[]> {
+    // @ts-ignore
```

Comment:
ts-ignore mire kell?

Reply:
Ezt a hibaüzenetet kapom:
 TS2322: Type 'Observable<string | Comment[]>' is not assignable to type 'Observable<Comment[]>'.   Type 'string | Comment[]' is not assignable to type 'Comment[]'.     Type 'string' is not assignable to type 'Comment[]'.

Reply:
lehet a komponensben használod ezt a típust, hogy Observable<string | Comment[]>